test/models/artefact_test.rb

```diff
@@ -251,13 +248,12 @@ class ArtefactTest < ActiveSupport::TestCase
     artefact.redirect_url = "/foobar#chapter"
     assert artefact.valid?
 
-    artefact.redirect_url = "http://foo.bar/"
-    refute artefact.valid?
+    artefact.redirect_url = "http://foo.bar.gov.uk/"
```

Comment:
Sorry I'm nitpicking - but I'd dare say schemas is the source of truth as Pub API should be following it https://github.com/alphagov/govuk-content-schemas/blob/master/dist/formats/redirect/publisher_v2/schema.json#L58-L67